examples/qg/run_qg.py

```diff
@@ -0,0 +1,210 @@
+from transformers import (
+    HfArgumentParser,
+    Seq2SeqTrainingArguments,
+    set_seed,
+)
+from primeqa.qg.processors.data_loader import QGDataLoader
+from dataclasses import dataclass,field
+from primeqa.qg.models.qg_model import QGModel
+from primeqa.qg.trainers.qg_trainer import QGTrainer
+from primeqa.qg.metrics.generation_metrics import rouge_metrics
+from primeqa.qg.utils.data_collator import T2TDataCollator
+from typing import Optional
+
+import json
+import logging
+import os
+import sys
+logger = logging.getLogger(__name__)
+
+@dataclass
+class ModelArguments:
+    """
+    Arguments pertaining to which model/config/tokenizer we are going to fine-tune from.
+    """
+
+    model_name_or_path: str = field(
+       default='t5-base', metadata={"help": "Path to pretrained model or model identifier from huggingface.co/models"}
+    )
+    modality: str = field(
+       default='table', metadata={"help": "If to work on tables or passages",
```

Comment:
This help description is awkward. Can you reword? Perhaps "Whether to generate questions from tables or passages"?